src/index.cpp

```diff
@@ -1424,7 +1424,11 @@ namespace diskann {
       copy_aligned_data_from_file<_u8>(pq_compressed_file.c_str(), _pq_data,
                                        file_num_points, _num_pq_chunks,
                                        _num_pq_chunks);
-      _pq_table.load_pq_centroid_bin(pq_pivots_file.c_str(), _num_pq_chunks);
+#ifdef EXEC_ENV_OLS
+      //pq_table.load_pq_centroid_bin(files, pq_pivots_file.c_str(), _num_pq_chunks);
```

Comment:
Can we throw an exception here for now, I think the behavior will be unexpected if pq_table is not  loaded